src/test/java/org/minions/devfund/aldo/WhackAMoleTest.java

```diff
@@ -0,0 +1,112 @@
+package org.minions.devfund.aldo;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * WhackAMole Test.
+ */
+public class WhackAMoleTest {
+    private WhackAMole whackAMole;
+
+    /**
+     * Constructor.
+     */
+    public WhackAMoleTest() {
+        final int numAttempts = 2;
+        final int gridDimension = 3;
+        whackAMole = new WhackAMole(numAttempts, gridDimension);
+        final int positionX = 0;
+        final int positionY = 0;
+        final int positionX1 = 1;
+        final int positionY1 = 1;
+        final int positionX2 = 2;
+        final int positionY2 = 2;
+        whackAMole.place(positionX, positionY);
+        whackAMole.place(positionX1, positionY1);
+        whackAMole.place(positionX2, positionY2);
+    }
+
+    /**
+     * printGridTest.
+     */
+    @Test
+    public void printGridTest() {
+        final int positionX = 0;
+        final int positionY = 0;
+        final int positionX1 = 1;
+        final int positionY1 = 1;
+        whackAMole.whack(positionX, positionY);
+        whackAMole.whack(positionX1, positionY1);
+        whackAMole.printGrid();
+    }
+
+    /**
+     * printGridToUser.
+     */
+    @Test
+    public void printGridToUser() {
+        final int positionX = 0;
+        final int positionY = 0;
+        final int positionX1 = 1;
+        final int positionY1 = 1;
+        whackAMole.whack(positionX, positionY);
+        whackAMole.whack(positionX1, positionY1);
+        whackAMole.printGridToUser();
+    }
+
+    /**
+     * placeTest.
+     */
+    @Test
+    public void placeTest() {
+        final int positionX = 0;
+        final int positionY = 0;
+        Assert.assertFalse(whackAMole.place(positionX, positionY));
+    }
+
+    /**
+     * placeMoleDecrement.
+     */
+    @Test
+    public void placeMoleDecrement() {
+        final int positionX = 1;
+        final int positionY = 0;
+        Assert.assertFalse(whackAMole.place(positionX, positionY));
+    }
+
+    /**
+     * whackTest1.
+     */
+    @Test
+    public void whackTest1() {
```

Comment:
Since your pull request is still open.
please write your unit tests something like this.

Running: testConstructorGrid
Finished: testConstructorGrid
Running: testConstructorNumOfAttempts
Finished: testConstructorNumOfAttempts
Running: testPlaceFail
Finished: testPlaceFail
Running: testWhackSucceedAttempts
Finished: testWhackSucceedAttempts
Running: testWhackFailAttempts
Finished: testWhackFailAttempts
Running: testPlaceSucceed
Finished: testPlaceSucceed
Running: testWhackFailMolesLeft
Finished: testWhackFailMolesLeft
Running: testWhackSucceedMolesLeft
Finished: testWhackSucceedMolesLeft